### ISSUE TITLE
Generate a flame graph for a post request

### ISSUE THREAD

**Issue by hraynaud** [SamSaffron/flamegraph#13]
Hi I was wondering if it was possible to generate the flamegraph on a post request?.

**Comment by SamSaffron** [comment]
we should add a way of triggering this in rack mini profiler its mainly a
UI problem, you can do this manually by writing middleware that detects the
request and then generates a graph to a file.

**Comment by hraynaud** [comment]
Hi Sam thanks for the quick response.  If you think it's not a huge undertaking I'm happy to take a crack at it if you can point me in the right direction.

**Comment by SamSaffron** [comment]
first step is a sample on how to do this in rack middleware that you can
add to a samples dir in flamegraph, you familiar with writing and inserting
rack middleware?

**Comment by hraynaud** [comment]
Yes I am. Ok let me see what I can put together over the weekend. Thanks

**Comment by davidpatters0n** [comment]
@hraynaud - Was there any movement on this ?
